keep-ui/entities/workflows/lib/__tests__/validation.test.ts

```diff
@@ -496,27 +496,6 @@ describe("validateGlobalPure", () => {
     ]);
   });
 
-  it("should detect empty alert trigger", () => {
```

Comment:
@talboren why did you remove this check? now it should check either cel or filters, right?

Reply:
Because empty alerts filter was OK even before. It meant to trigger every alert.